docs/melange-book/melange-playground.md

```diff
@@ -0,0 +1,106 @@
+# Melange Playground
+
+[Melange Playground](https://melange.re/unstable/playground/) is an interactive
+environment for running code and seeing its output, including the JavaScript
+generated by Melange compiler. It's also a great way to share code
+snippets--the URL can be copied and pasted into another tab, recreating the
+exact text inside the code editor.
+
+In the next section, we'll use the Melange Playground to explore OCaml's numeric
+types.
+
+## OCaml numeric types
+
+Unlike JavaScript, OCaml makes a clear and hard distinction between integer and
+float values. Try running the following program in the playground:
+
+```reasonml
+let foo = 42;   // int
+let bar = 42.0; // float
+Js.log(foo == bar);
+```
+
+You'll get a compiler error for the last line that says
+
+```
+This expression has type float but an expression was expected of type int
+```
+
+Unlike JavaScript, there are no implicit conversions in OCaml. Therefore you
+cannot expect to compare an integer and a float together, unless you convert one
+of the values so that both values have the same type. To make the last line
+compile, you can change it to
+
+```reasonml
+Js.log(foo == Int.of_float(bar));
+```
+
+Another way to fix the last line is to convert `foo` from an int to a float:
+
+```reasonml
+Js.log(Float.of_int(foo) == bar);
+```
+
+What about addition? From what you've already seen, you can probably guess that
+`Js.log(42 + 16.0)` won't compile. However, you may be surprised to discover
+that `Js.log(42.0 + 16.0)` also won't compile! That's because OCaml uses
+separate arithmetic operators for floats. What will compile is this:
+
+```reasonml
+Js.log(42.0 +. 16.0);
+```
+
+Underneath the covers, `foo` and `bar` are both instances of JavaScript's `Number
+type`:
+
+```reasonml
+Js.log(Js.typeof(foo)); // prints "number"
+Js.log(Js.typeof(bar)); // prints "number"
+```
+
+Refer to the [Melange
+docs](../communicate-with-javascript.md#data-types-and-runtime-representation)
+for a complete rundown of how OCaml types get translated to JavaScript types.
+
+## Exercises
+
+1. Convert the Counter component we created in the previous chapter to use float
+   instead of integer. Make the `-` button decrement by `0.5` and the `+` button
+   increment by `1.5`.
+1. Add an int64 value to your program:
+   ```reasonml
+   let baz = 42_000_000_000L; // int64
+   Js.log(baz);
+   ```
```

Comment:
Unfortunately, mkdocs [doesn't support indented code blocks](https://github.com/mkdocs/mkdocs/issues/282), so this looks bad atm:

<img width="704" alt="imagen" src="https://github.com/melange-re/melange-re.github.io/assets/220424/ee2a90c2-d67f-4eae-a7a0-6f29bd667aaa">

We can fix it by using indented code block, at the expense of losing syntax highlighting:

```suggestion
   
        let baz = 42_000_000_000L; // int64
        Js.log(baz);
```

Reply:
Fixed in https://github.com/melange-re/melange-re.github.io/pull/93/commits/d191829f5cbb424b6e9c8122fbd0b936e2fc369a